monitor/api/src/main/java/org/datacleaner/monitor/scheduling/model/ScheduleDefinitionModel.java

```diff
@@ -24,6 +24,9 @@
 
 public class ScheduleDefinitionModel {
     private final String _hotFolder;
+    
+    @JsonProperty("exportMetrics")
+    private boolean _exportMetrics;
 
     @JsonCreator
     public ScheduleDefinitionModel(@JsonProperty("hotFolder") final String hotFolder) {
```

Comment:
Maybe we should also move this `@JsonProperty` to a setter? I'd prefer this an immutable object, but as long as we're consistent, both is fine with me.